l5kit/l5kit/geometry/transform.py

```diff
@@ -70,47 +70,74 @@ def flip_y_axis(tm: np.ndarray, y_dim_size: int) -> np.ndarray:
     return tm
 
 
+def transform_points_batch(points: np.ndarray, transf_matrix: np.ndarray) -> np.ndarray:
```

Comment:
@lucabergamini sorry for the super late review, can we combine `transform_points_batch ` with `transform_points`? Just use a single interface

Reply:
done :)